app/src/main/java/com/vivek/bodyscan/ui/components/ScanningCanvas.kt

```diff
@@ -37,14 +37,24 @@ fun ScanningCanvas(
     modifier: Modifier
 ) {
     var scannedOffset by remember { mutableStateOf(Offset(-500f, -500f)) }
+    var circleRadius by remember { mutableStateOf(300f) }
 
     Canvas(
         modifier = modifier
             .fillMaxSize()
             .background(OverlayScreenBGColor)
             .padding(4.dp)
             .pointerInput(Unit) {
-                detectDragGestures { change, _ ->
+                detectDragGestures(
+                    onDragStart = { offset ->
+                        scannedOffset = offset
```

Comment:
I reviewed it and found that `this line is unnecessary` as we are changing the scanned offset in the `lambda function` also

Reply:
@V9vek I've updated to code to remove the unnecessary assignment

Reply:
okay thanks for the changes